### DISCUSSION TITLE
BNT 3

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#481] [Show and tell]
### Channel ID

BNT3.bg

### Stream URL (optional)

https://cdn3.gledam.xyz/hls/hd-bnt-3-hd/index.m3u8?e=1692278011&hash=RVh_cg10XM0nowcY4C3lNw

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://cdn3.gledam.xyz/hls/hd-bnt-3-hd/index.m3u8?e=1692278011&hash=RVh_cg10XM0nowcY4C3lNw (offline)
